pkg/ctl/utils/enable_logging.go

```diff
@@ -0,0 +1,148 @@
+package utils
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/kris-nova/logger"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	"github.com/weaveworks/eksctl/pkg/ctl/cmdutils"
+	"github.com/weaveworks/eksctl/pkg/eks"
+	"github.com/weaveworks/eksctl/pkg/printers"
+)
+
+func enableLoggingCmd(rc *cmdutils.ResourceCmd) {
+	cfg := api.NewClusterConfig()
+	rc.ClusterConfig = cfg
+
+	rc.SetDescription("enable-logging", "Update cluster logging configuration", "")
+
+	rc.SetRunFuncWithNameArg(func() error {
+		return doEnableLogging(rc)
+	})
+
+	rc.FlagSetGroup.InFlagSet("General", func(fs *pflag.FlagSet) {
+		cmdutils.AddNameFlag(fs, cfg.Metadata)
+		cmdutils.AddRegionFlag(fs, rc.ProviderConfig)
+		cmdutils.AddConfigFileFlag(fs, &rc.ClusterConfigFile)
+		cmdutils.AddApproveFlag(fs, rc)
+	})
+
+	rc.FlagSetGroup.InFlagSet("Enable/disable log types", func(fs *pflag.FlagSet) {
+		enableExplicitly := sets.NewString()
+		allSupportedTypes := api.SupportedCloudWatchClusterLogTypes()
+		inherit := sets.NewString()
+
+		enableAll := fs.Bool("all", true, fmt.Sprintf("Enable all supported log types (%s)", strings.Join(allSupportedTypes, ", ")))
+
+		for _, logType := range allSupportedTypes {
+			_ = fs.Bool(logType, false, fmt.Sprintf("Enable %q log type", logType))
```

Comment:
I find the camel case naming for the `--controllerManager` flag inconsistent with other flags which are hyphen-separated, but I can see it saves us some code.

For consistency, WDYT about handling this differently at the expense of repeating the supported logging options here?

Reply:
I did think about this, but didn't want to make it complicated... feel free to implement if you have ideas.

In away I'm also struggling with the fact that it's `controllerManager` and not `ControllerManager`, as we are trying to make string constants adhere to Kubernetes-style (#1065). So far it's seems like our constants will be Kubernetes-style, with an exception for the ones defined by AWS.